{"uid":"6e35f9d84c4924d1","name":"TC 02.02.05 - Check Contact Us link in footer leads to the correct page from each page specified in DATA_1","fullName":"tests.test_footer.TestFooter#test_tc_02_02_05_check_contact_us_link_functionality","historyId":"94f0e1cf98dd00a5517d7dafd8a82281","time":{"start":1708391540152,"stop":1708391546115,"duration":5963},"description":"Check that Contact Us link in footer is correct","descriptionHtml":"<p>Check that Contact Us link in footer is correct</p>\n","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x564b4eab9d93 <unknown>\n#1 0x564b4e79d337 <unknown>\n#2 0x564b4e7e8d60 <unknown>\n#3 0x564b4e7e8e51 <unknown>\n#4 0x564b4e82cbc4 <unknown>\n#5 0x564b4e80b46d <unknown>\n#6 0x564b4e82a12d <unknown>\n#7 0x564b4e80b1e3 <unknown>\n#8 0x564b4e7db135 <unknown>\n#9 0x564b4e7dc13e <unknown>\n#10 0x564b4ea7de4b <unknown>\n#11 0x564b4ea81dfa <unknown>\n#12 0x564b4ea6a6d5 <unknown>\n#13 0x564b4ea82a6f <unknown>\n#14 0x564b4ea4e69f <unknown>\n#15 0x564b4eaa7098 <unknown>\n#16 0x564b4eaa7262 <unknown>\n#17 0x564b4eab8f34 <unknown>\n#18 0x7f0871094ac3 <unknown>","statusTrace":"self = <test_footer.TestFooter object at 0x7f9b31c935d0>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"9e48b91e0b6dd50ee420b908eb582924\")>\nURL = 'https://magento.softwaretestingboard.com/what-is-new.html'\n\n    @allure.title(\"TC 02.02.05 - Check Contact Us link in footer leads to the correct page \"\n                  \"from each page specified in DATA_1\")\n    @pytest.mark.parametrize('URL', DATA_1)\n    def test_tc_02_02_05_check_contact_us_link_functionality(self, driver, URL):\n        \"\"\"Check that Contact Us link in footer is correct\"\"\"\n        page = FooterPage(driver, url=URL)\n        page.open()\n>       page.check_contact_us_link_functionality()\n\ntests/test_footer.py:557: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/footer_page.py:326: in check_contact_us_link_functionality\n    contact_us_link = self.element_is_visible(self.footer_locators.CONTACT_US_LINK)\npages/base_page.py:29: in element_is_visible\n    self.go_to_element(self.element_is_present(locator))\npages/base_page.py:50: in element_is_present\n    return wait(self.driver, timeout).until(EC.presence_of_element_located(locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"9e48b91e0b6dd50ee420b908eb582924\")>\nmethod = <function presence_of_element_located.<locals>._predicate at 0x7f9b31a3fb00>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x564b4eab9d93 <unknown>\nE       #1 0x564b4e79d337 <unknown>\nE       #2 0x564b4e7e8d60 <unknown>\nE       #3 0x564b4e7e8e51 <unknown>\nE       #4 0x564b4e82cbc4 <unknown>\nE       #5 0x564b4e80b46d <unknown>\nE       #6 0x564b4e82a12d <unknown>\nE       #7 0x564b4e80b1e3 <unknown>\nE       #8 0x564b4e7db135 <unknown>\nE       #9 0x564b4e7dc13e <unknown>\nE       #10 0x564b4ea7de4b <unknown>\nE       #11 0x564b4ea81dfa <unknown>\nE       #12 0x564b4ea6a6d5 <unknown>\nE       #13 0x564b4ea82a6f <unknown>\nE       #14 0x564b4ea4e69f <unknown>\nE       #15 0x564b4eaa7098 <unknown>\nE       #16 0x564b4eaa7262 <unknown>\nE       #17 0x564b4eab8f34 <unknown>\nE       #18 0x7f0871094ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.11.8/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_session_faker","time":{"start":1708391232614,"stop":1708391232678,"duration":64},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"clear_allure_result_folder","time":{"start":1708391232678,"stop":1708391232678,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"driver","time":{"start":1708391539196,"stop":1708391540152,"duration":956},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"description":"Check that Contact Us link in footer is correct","status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x564b4eab9d93 <unknown>\n#1 0x564b4e79d337 <unknown>\n#2 0x564b4e7e8d60 <unknown>\n#3 0x564b4e7e8e51 <unknown>\n#4 0x564b4e82cbc4 <unknown>\n#5 0x564b4e80b46d <unknown>\n#6 0x564b4e82a12d <unknown>\n#7 0x564b4e80b1e3 <unknown>\n#8 0x564b4e7db135 <unknown>\n#9 0x564b4e7dc13e <unknown>\n#10 0x564b4ea7de4b <unknown>\n#11 0x564b4ea81dfa <unknown>\n#12 0x564b4ea6a6d5 <unknown>\n#13 0x564b4ea82a6f <unknown>\n#14 0x564b4ea4e69f <unknown>\n#15 0x564b4eaa7098 <unknown>\n#16 0x564b4eaa7262 <unknown>\n#17 0x564b4eab8f34 <unknown>\n#18 0x7f0871094ac3 <unknown>","statusTrace":"self = <test_footer.TestFooter object at 0x7f9b31c935d0>\ndriver = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"9e48b91e0b6dd50ee420b908eb582924\")>\nURL = 'https://magento.softwaretestingboard.com/what-is-new.html'\n\n    @allure.title(\"TC 02.02.05 - Check Contact Us link in footer leads to the correct page \"\n                  \"from each page specified in DATA_1\")\n    @pytest.mark.parametrize('URL', DATA_1)\n    def test_tc_02_02_05_check_contact_us_link_functionality(self, driver, URL):\n        \"\"\"Check that Contact Us link in footer is correct\"\"\"\n        page = FooterPage(driver, url=URL)\n        page.open()\n>       page.check_contact_us_link_functionality()\n\ntests/test_footer.py:557: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \npages/footer_page.py:326: in check_contact_us_link_functionality\n    contact_us_link = self.element_is_visible(self.footer_locators.CONTACT_US_LINK)\npages/base_page.py:29: in element_is_visible\n    self.go_to_element(self.element_is_present(locator))\npages/base_page.py:50: in element_is_present\n    return wait(self.driver, timeout).until(EC.presence_of_element_located(locator))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"9e48b91e0b6dd50ee420b908eb582924\")>\nmethod = <function presence_of_element_located.<locals>._predicate at 0x7f9b31a3fb00>\nmessage = ''\n\n    def until(self, method, message: str = \"\"):\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message: \nE       Stacktrace:\nE       #0 0x564b4eab9d93 <unknown>\nE       #1 0x564b4e79d337 <unknown>\nE       #2 0x564b4e7e8d60 <unknown>\nE       #3 0x564b4e7e8e51 <unknown>\nE       #4 0x564b4e82cbc4 <unknown>\nE       #5 0x564b4e80b46d <unknown>\nE       #6 0x564b4e82a12d <unknown>\nE       #7 0x564b4e80b1e3 <unknown>\nE       #8 0x564b4e7db135 <unknown>\nE       #9 0x564b4e7dc13e <unknown>\nE       #10 0x564b4ea7de4b <unknown>\nE       #11 0x564b4ea81dfa <unknown>\nE       #12 0x564b4ea6a6d5 <unknown>\nE       #13 0x564b4ea82a6f <unknown>\nE       #14 0x564b4ea4e69f <unknown>\nE       #15 0x564b4eaa7098 <unknown>\nE       #16 0x564b4eaa7262 <unknown>\nE       #17 0x564b4eab8f34 <unknown>\nE       #18 0x7f0871094ac3 <unknown>\n\n/opt/hostedtoolcache/Python/3.11.8/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py:95: TimeoutException","steps":[{"name":"Open a browser","time":{"start":1708391540152,"stop":1708391541028,"duration":876},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"Check Contact Us link leads to the correct page","time":{"start":1708391541029,"stop":1708391546114,"duration":5085},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x564b4eab9d93 <unknown>\n#1 0x564b4e79d337 <unknown>\n#2 0x564b4e7e8d60 <unknown>\n#3 0x564b4e7e8e51 <unknown>\n#4 0x564b4e82cbc4 <unknown>\n#5 0x564b4e80b46d <unknown>\n#6 0x564b4e82a12d <unknown>\n#7 0x564b4e80b1e3 <unknown>\n#8 0x564b4e7db135 <unknown>\n#9 0x564b4e7dc13e <unknown>\n#10 0x564b4ea7de4b <unknown>\n#11 0x564b4ea81dfa <unknown>\n#12 0x564b4ea6a6d5 <unknown>\n#13 0x564b4ea82a6f <unknown>\n#14 0x564b4ea4e69f <unknown>\n#15 0x564b4eaa7098 <unknown>\n#16 0x564b4eaa7262 <unknown>\n#17 0x564b4eab8f34 <unknown>\n#18 0x7f0871094ac3 <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.8/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/footer_page.py\", line 326, in check_contact_us_link_functionality\n    contact_us_link = self.element_is_visible(self.footer_locators.CONTACT_US_LINK)\n                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.8/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 29, in element_is_visible\n    self.go_to_element(self.element_is_present(locator))\n                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.8/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 50, in element_is_present\n    return wait(self.driver, timeout).until(EC.presence_of_element_located(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.8/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[{"name":"Find a visible element","time":{"start":1708391541029,"stop":1708391546114,"duration":5085},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x564b4eab9d93 <unknown>\n#1 0x564b4e79d337 <unknown>\n#2 0x564b4e7e8d60 <unknown>\n#3 0x564b4e7e8e51 <unknown>\n#4 0x564b4e82cbc4 <unknown>\n#5 0x564b4e80b46d <unknown>\n#6 0x564b4e82a12d <unknown>\n#7 0x564b4e80b1e3 <unknown>\n#8 0x564b4e7db135 <unknown>\n#9 0x564b4e7dc13e <unknown>\n#10 0x564b4ea7de4b <unknown>\n#11 0x564b4ea81dfa <unknown>\n#12 0x564b4ea6a6d5 <unknown>\n#13 0x564b4ea82a6f <unknown>\n#14 0x564b4ea4e69f <unknown>\n#15 0x564b4eaa7098 <unknown>\n#16 0x564b4eaa7262 <unknown>\n#17 0x564b4eab8f34 <unknown>\n#18 0x7f0871094ac3 <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.8/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 29, in element_is_visible\n    self.go_to_element(self.element_is_present(locator))\n                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.8/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 50, in element_is_present\n    return wait(self.driver, timeout).until(EC.presence_of_element_located(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.8/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[{"name":"Find a present element","time":{"start":1708391541029,"stop":1708391546114,"duration":5085},"status":"broken","statusMessage":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x564b4eab9d93 <unknown>\n#1 0x564b4e79d337 <unknown>\n#2 0x564b4e7e8d60 <unknown>\n#3 0x564b4e7e8e51 <unknown>\n#4 0x564b4e82cbc4 <unknown>\n#5 0x564b4e80b46d <unknown>\n#6 0x564b4e82a12d <unknown>\n#7 0x564b4e80b1e3 <unknown>\n#8 0x564b4e7db135 <unknown>\n#9 0x564b4e7dc13e <unknown>\n#10 0x564b4ea7de4b <unknown>\n#11 0x564b4ea81dfa <unknown>\n#12 0x564b4ea6a6d5 <unknown>\n#13 0x564b4ea82a6f <unknown>\n#14 0x564b4ea4e69f <unknown>\n#15 0x564b4eaa7098 <unknown>\n#16 0x564b4eaa7262 <unknown>\n#17 0x564b4eab8f34 <unknown>\n#18 0x7f0871094ac3 <unknown>\n\n","statusTrace":"  File \"/opt/hostedtoolcache/Python/3.11.8/x64/lib/python3.11/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"/home/runner/work/luma_project/luma_project/pages/base_page.py\", line 50, in element_is_present\n    return wait(self.driver, timeout).until(EC.presence_of_element_located(locator))\n           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"/opt/hostedtoolcache/Python/3.11.8/x64/lib/python3.11/site-packages/selenium/webdriver/support/wait.py\", line 95, in until\n    raise TimeoutException(message, screen, stacktrace)\n","steps":[],"attachments":[],"parameters":[{"name":"locator","value":"('css selector', '.footer div :nth-child(3) a')"},{"name":"timeout","value":"5"}],"shouldDisplayMessage":true,"stepsCount":0,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[{"name":"locator","value":"('css selector', '.footer div :nth-child(3) a')"},{"name":"timeout","value":"10"}],"shouldDisplayMessage":false,"stepsCount":1,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":2,"attachmentsCount":0,"hasContent":true,"attachmentStep":false}],"attachments":[{"uid":"36df19ad3cae0bae","name":"Screenshot","source":"36df19ad3cae0bae.png","type":"image/png","size":545595},{"uid":"210f0b835a6c5c7c","name":"HTML source","source":"210f0b835a6c5c7c.html","type":"text/html","size":115140}],"parameters":[],"shouldDisplayMessage":true,"stepsCount":4,"attachmentsCount":2,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"driver::0","time":{"start":1708391546313,"stop":1708391546380,"duration":67},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"clear_allure_result_folder::0","time":{"start":1708392514897,"stop":1708392514897,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"epic","value":"Test Footer"},{"name":"parentSuite","value":"tests"},{"name":"suite","value":"test_footer"},{"name":"subSuite","value":"TestFooter"},{"name":"host","value":"fv-az738-374"},{"name":"thread","value":"2957-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.test_footer"},{"name":"resultFormat","value":"allure2"}],"parameters":[{"name":"URL","value":"'https://magento.softwaretestingboard.com/what-is-new.html'"}],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":3,"skipped":0,"passed":0,"unknown":0,"total":3},"items":[{"uid":"6cb56b072823df34","reportUrl":"https://verafes.github.io/luma_project/225//#testresult/6cb56b072823df34","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x55cbfb0d9d93 <unknown>\n#1 0x55cbfadbd337 <unknown>\n#2 0x55cbfae08d60 <unknown>\n#3 0x55cbfae08e51 <unknown>\n#4 0x55cbfae4cbc4 <unknown>\n#5 0x55cbfae2b46d <unknown>\n#6 0x55cbfae4a12d <unknown>\n#7 0x55cbfae2b1e3 <unknown>\n#8 0x55cbfadfb135 <unknown>\n#9 0x55cbfadfc13e <unknown>\n#10 0x55cbfb09de4b <unknown>\n#11 0x55cbfb0a1dfa <unknown>\n#12 0x55cbfb08a6d5 <unknown>\n#13 0x55cbfb0a2a6f <unknown>\n#14 0x55cbfb06e69f <unknown>\n#15 0x55cbfb0c7098 <unknown>\n#16 0x55cbfb0c7262 <unknown>\n#17 0x55cbfb0d8f34 <unknown>\n#18 0x7f5c68c94ac3 <unknown>","time":{"start":1708305291395,"stop":1708305297283,"duration":5888}},{"uid":"ef4b5aecd4337e43","reportUrl":"https://verafes.github.io/luma_project/224//#testresult/ef4b5aecd4337e43","status":"broken","statusDetails":"selenium.common.exceptions.TimeoutException: Message: \nStacktrace:\n#0 0x557e23ed1d93 <unknown>\n#1 0x557e23bb5337 <unknown>\n#2 0x557e23c00d60 <unknown>\n#3 0x557e23c00e51 <unknown>\n#4 0x557e23c44bc4 <unknown>\n#5 0x557e23c2346d <unknown>\n#6 0x557e23c4212d <unknown>\n#7 0x557e23c231e3 <unknown>\n#8 0x557e23bf3135 <unknown>\n#9 0x557e23bf413e <unknown>\n#10 0x557e23e95e4b <unknown>\n#11 0x557e23e99dfa <unknown>\n#12 0x557e23e826d5 <unknown>\n#13 0x557e23e9aa6f <unknown>\n#14 0x557e23e6669f <unknown>\n#15 0x557e23ebf098 <unknown>\n#16 0x557e23ebf262 <unknown>\n#17 0x557e23ed0f34 <unknown>\n#18 0x7fb8de894ac3 <unknown>","time":{"start":1708219057076,"stop":1708219063001,"duration":5925}}]},"tags":[]},"source":"6e35f9d84c4924d1.json","parameterValues":["'https://magento.softwaretestingboard.com/what-is-new.html'"]}